base_model_metaclass_mixin/__init__.py

```diff
@@ -0,0 +1,33 @@
+from openerp.osv import orm
+from openerp.osv import osv
+from openerp.tools.translate import _
+
+__all__ = ['BaseModelMetaclassMixin']
+
+
+def get_overrides():
+    overrides = {}
```

Comment:
this will be global for all databases running on the server. Shouldn't this be per database (as in: Set up the data structure in `_register_hook` and keep it per registry)?